site/docs/uk/guide/inline-queries.md

```diff
@@ -0,0 +1,64 @@
+---
+prev: ./errors.md
+next: ./files.md
+---
+
+# Inline запити
+
+За допомогою inline запитів користувачі можуть шукати, переглядати та надсилати контент, запропонований вашим ботом, у будь-якому чаті, навіть якщо бот не учасник цього чату.
+Для цього користувачі починають повідомлення з `@your_bot_name` і вибирають один із результатів.
+
+::: tip Увімкніть inline режим
+За замовчуванням inline режим вимкнено.
+Зверніться до [@BotFather](https://t.me/BotFather), щоб увімкнути inline режим для свого бота, щоб почати отримувати inline запити.
```

Comment:
```suggestion
Для того, щоб отримувати inline запити, потрібно звернутися до [@BotFather](https://t.me/BotFather)(https://t.me/BotFather) та увімкнути inline режим для свого бота.
```

I am making this suggestion because the word "щоб" is used too many times in the sentence...